Misc/NEWS.d/next/Library/2018-04-18-16-15-55.bpo-24905.jYqjYx.rst

```diff
@@ -0,0 +1,4 @@
+The :class:`sqlite3.Connection` now has the
+:meth:`sqlite3.Connection.open_blob` method. The :class:`sqlite3.Blob`
+allows incremental I/O operations to blobs. (Contributed by Aviv Palivoda in
```

Comment:
It would be nice to add a whatsnew entry. We can use the following attribute format there:

```rst
(Contributed by Aviv Palivoda in :issue:`24905`.)
```

Reply:
Should I add the whatsnew entry?